src/main/app/controller/courts/ContactsController.ts

```diff
@@ -34,6 +34,10 @@ export class ContactsController {
       .then((value: ContactType[]) => types = value)
       .catch(() => error += this.getContactTypesErrorMsg);
 
+    if (!contacts?.some(c => c.isNew === true)) {
```

Comment:
A small one so apologies if its silly

(!contacts?.some(c => c.isNew === true))

Will (!contacts?.some(c => c.isNew)) work on its own?

Reply:
I'm using this format to avoid any other 'truthy' value causing this to resolve to true.